Xamarin.Forms.Core/Internals/InvalidationTrigger.cs

```diff
@@ -1,9 +1,9 @@
 using System;
 
-namespace Xamarin.Forms
+namespace Xamarin.Forms.Internals
 {
 	[Flags]
-	internal enum InvalidationTrigger
+	public enum InvalidationTrigger
```

Comment:
`InvalidationTrigger` is the parameter type for `IVisualElementController.InvalidateMeasure`, which is called from several renderers in Platform projects. Similar situation to `EvalRequested`, `Ticker`, `IDataTemplate`, etc.